docs/release-notes/v1.6.0.adoc

```diff
@@ -0,0 +1,8 @@
+= v1.5.0
+
+All images are 1:1 compatible with version 1.5.0.
+
+.Changes
+* The *builder-dind-runner* builder image introduced:
```

Comment:
Elég ha a dokumentációban rámutatsz az új image leírására és egyértelmű lesz, és nem is kell duplán írnod semmit :)